docs/developers-clear-containers-install.md

```diff
@@ -124,13 +124,41 @@ See [the upgrading document](upgrading.md) for further details.
    $ sudo sed -i.bak -e 's!^\(image = ".*"\)!# \1 \
    image = "/usr/share/clear-containers/container.img"!g' \
    /usr/share/defaults/clear-containers/configuration.toml
-   
+   ```
 For more details on the runtime's build system, run:
 
 ```bash
 $ make help
 ```
 
+5. Kernel
+
+   The kernel latest kernel for Clear Linux can be found in
+   https://github.com/clearcontainers/linux/releases, It can be used to rebuild
+   and modify it as need. The example below demonstrates how to do this using the
+   osbuilder tooling.
+
+   ```bash
+	$ cd $GOPATH/src/github.com/clearcontainers/osbuilder
+
+	$ # Fetch latest kernel sources from github.com/clearcontainers/linux
+	$ make make kernel-src
+
+	$ # Optional modify kernel sources or config at $PWD/workdir/linux
+
+	$ # build kernel
+	$ make kernel
+
+	$ # Install the custom image
+	$ sudo install --owner root --group root --mode 0755 workdir/vmlinuz.container /usr/share/clear-containers/custom-vmlinuz
+	$ sudo install --owner root --group root --mode 0755 workdir/vmlinux.container /usr/share/clear-containers/custom-vmlinux
+
+	$ # Update the runtime configuration
+	$ # (note that this is only an example using default paths).
+	$ sudo sed -i.bak -e 's!^\(kernel = ".*"\)!# \1\nkernel = "/usr/share/clear-containers/custom-vmlinuz"!g' \
+	/usr/share/defaults/clear-containers/configuration.toml
+	 ```
```

Comment:
We need to remember to add details for vmlinux too once we switch back from pc platform type.

Reply:
@mcastelino added a note.